packages-ts/gauntlet-cosmos-contracts/src/commands/contracts/ocr2/close/acceptProposal.ts

```diff
@@ -12,9 +12,14 @@ const makeInput = async (flags): Promise<AcceptProposalInput> => {
   }
   if (flags.input) return { ...flags.input, ...defaultInput }
 
+  if (!process.env.SECRET) {
```

Comment:
had to add this because I changed [packages-ts/gauntlet-cosmos-contracts/src/commands/contracts/ocr2/proposal/acceptProposal.ts](https://github.com/smartcontractkit/chainlink-cosmos/pull/364/files#diff-4adab7e87a1782cdf7257c950b247df53842ede9b054aba954939b83c7aad5b2)